pages/01.Home/02.Contributing/01.Style Guide/docs.en.md

```diff
@@ -0,0 +1,289 @@
+    {
+        defined('MAUTIC_ENV') or define('MAUTIC_ENV', $environment);
+        defined('MAUTIC_VERSION') or define(
+            'MAUTIC_VERSION',
+            self::MAJOR_VERSION.'.'.self::MINOR_VERSION.'.'.self::PATCH_VERSION.self::EXTRA_VERSION
+        );
+
+        parent::__construct($environment, $debug);
+    }
+    ```
+
+1. END list item
+
+<details><summary>markdown syntax</summary>
+
+##### Nested `<codeblock>` markdown syntax
+
+```markdown
+    ```php
+    /**
+     * Constructor.
+     *
+     * @param string $environment The environment
+     * @param bool   $debug       Whether to enable debugging or not
+     *
+     * @api
+     */
+    public function __construct($environment, $debug)
+    {
+        defined('MAUTIC_ENV') or define('MAUTIC_ENV', $environment);
+        defined('MAUTIC_VERSION') or define(
+            'MAUTIC_VERSION',
+            self::MAJOR_VERSION.'.'.self::MINOR_VERSION.'.'.self::PATCH_VERSION.self::EXTRA_VERSION
+        );
+
+        parent::__construct($environment, $debug);
+    }
+    ```
+```
+
+</details>
+</details>
+
+### Links
+
+[link macro]: <http://example.com>
+[testing]: <./../plugins/integration_test.html>
+
+Often you'll want to make a link to another place in the documentation.  We prefer to group links at the bottom of a page, and provide a reference macro in the text.
+This make linking to the same place much easier.  In Markdown, it looks like this:
+
+#### Absolute links
+
+You can have different links in your text, some leading [here][link macro] and others leading [there][link macro].
+
+```markdown
+You can have different links in your text, some leading [here][link macro] and others leading [there][link macro].
+
+[link macro]: <http://example.com>
+```
+
+#### Relative links
+
+These will link to [`plugins/integration_test.html`][testing] on the documentation website created from the [_`plugins/integration_test.md`_][testing] source file.
+
+The first line is an example from within the same section.  The second for anywhere else in the directory structure.
+
+```markdown
+[testing]: <integration_test.html>
+
+[testing]: <./../plugins/integration_test.html>
+```
+
+#### Heading anchors
+
+Heading anchors enable linking directly to a Markdown heading from within the same document very easily.  The anchors are auto-generated for all headings.
+The link target is specified inline.
+
+```markdown
+You can have different anchors in your text, for example to this [Section][#anchor-section] right here.
+
+### Images
+
+Images should be placed in the media subfolders in the different sections of this repository.
+
+Link format:
+
+```markdown
+![alternative text here](image-reference.png "Tooltip text here")
+```
+
+#### Relative image links
+
+To display an image already in the documentation repository, use a relative path.
+
+The first line is an example of using images in the same section.  The second for anywhere else in the directory structure.
```

Comment:
Please check if this is still this case - it was in our old Gitbook docs but I do not know if that is still the case now.

Reply:
I tried using a relative link to link to an image somewhere else in the directory structure. It didn't work. So, removed this topic.